code/go/ecs/tracing.go

```diff
@@ -34,4 +34,9 @@ type Tracing struct {
 	// A transaction is the highest level of work measured within a service,
 	// such as a request to a server.
 	TransactionID string `ecs:"transaction.id"`
+
+	// Unique identifier of the span.
```

Comment:
nit: should we rephrase to _Unique identifier of the span per trace_?

Reply:
I'd be happy to change `transaction.id` and `span.id` descriptions to read

> Unique identifier of the transaction|span within the scope of its trace.

@felixbarny WDYT?

Reply:
Nice catch Silvia, SGTM

Reply:
Updated, thanks for the suggestion.